### DISCUSSION TITLE
Question regarding the app reaching out to other servers?

### DISCUSSION THREAD

**Discussion by Rigel09** [giejay/Immich-Android-TV#22] [Q&A] [answered]
Looks like an awesome idea, thanks for the app!

Question about which servers this app talks to? Does this app reach out to other servers other than my Immich instance? I assumed it would just talk to Immich? I downloaded the app and noticed it listed a server QR code along with a separate URL in order to connect to my Immich instance. What is this URL for? Is it only for linking to my Immich instance?

Note: I did notice the option to manually type a URL as well but was still curious about who all the app was talking to.

Thanks again for the app contribution!

- Rigel

**Comment by giejay** [comment, marked as answer]
Hi, the other url, immich.giejay.nl is a simple node project to be able to link your immich instance to the Android TV app. You don't need to use it, you can indeed manually type it as well. The Android TV app itself only talks to your Immich instance.
